### PR TITLE
Cleanup naming of Kinesis to be more consistent.

### PR DESCRIPTION
**Pull request by Integralist**
The naming of our Kinesis public functions are inconsistent and in some cases incorrect. This PR addresses the various references to Kinesis.

### CODE REVIEW

**Review by phamann — APPROVED** [review]
👍🏻 LGTM

File: fastly/kinesis.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ type Kinesis struct {
 	DeletedAt         *time.Time `mapstructure:"deleted_at"`
 }
 
-// kinesesByName is a sortable list of Kineses.
+// kinesesByName is a sortable list of Kinesis.
```
Comment (doramatadora) [inline fastly/kinesis.go:29]:
```suggestion
// kinesisByName is a sortable list of Kinesis.
```
And in the type below :)

**Review by doramatadora — CHANGES_REQUESTED** [review]
I think you missed one, otherwise LGTM

File: fastly/kinesis.go (hunk 1 above)
Comment (Integralist) [reply fastly/kinesis.go:29]: Nice spot!

**Review by doramatadora — APPROVED** [review]
image

**Pull request merged**